jdaviz/configs/default/plugins/model_fitting/tests/test_fitting.py

```diff
@@ -119,14 +113,16 @@ def test_cube_fitting_backend():
 
     SIGMA = 0.1  # noise in data
     TOL = 0.4  # test tolerance
+    IMAGE_SIZE_X = 15
```

Comment:
I purposely made this asymmetric so it is easier to tell if we get the spatial dimension wrong.